06/index.html

```diff
@@ -0,0 +1,115 @@
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Togemon</li>
+        <img src="./assets/Togemon.webp" width="100">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Patamon</li>
+        <img src="./assets/Patamon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Gabumon</li>
+        <img src="./assets/Gabumon.webp" width="100">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>WereGarurumon</li>
+        <img src="./assets/WereGarurumon.webp" width="100">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Angewomon</li>
+        <img src="./assets/Angewomon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Gomamon</li>
+        <img src="./assets/Gomamon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Zudomon</li>
+        <img src="./assets/Zudomon.webp" width="150">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+      </ol>
+    </nav>
+    <aside>
+      <h2>Opening</h2>
+      <video src="./assets/Digimon Adventure 01 Opening Español Latino (720p).mp4" controls poster="./assets/digimon miniatura.jpeg"></video>
+    </aside>
+    <article>
+      <h2>Demo</h2>
+      <iframe src="https://digimon.fandom.com/es/wiki/Digimon_Adventure" width="900" height="600"></iframe>
+    </article>
+    <section>
+      <from>
+        <h3>Acerca de ti</h3>
+        <fieldset>
+          <span>Nombre:</span>
+          <br>
+          <input type="text" value="Nombre">
+          <br>
+          <br>
+          <fieldset>
+            <legend>¿Sabes sobre pokemon?</legend>
+            <label for="Si">Si</label>
+            <input type="radio" id="Si" name="decision">
+            <br>
+            <label for="No">No</label>
+            <input type="radio" id="No" name="decision">
+          </fieldset>
+          <br>
+          <label for="digimon">Selecióna tu pokemon favorito</label>
+          <br>
+          <select id="digimon" name="digimon">
+            <option>--Selecciona un pokemon--</option>
+            <option value="Agumon">Agumon</option>
+            <option value="Angemon">Angemon</option>
+            <option value="Angewomon">Angewomon</option>
+            <option value="Gabumon">Gabumon</option>
+            <option value="Gomamon">Gomamon</option>
+            <option value="Greymon">Greymon</option>
+            <option value="Patamon">Patamon</option>
+            <option value="Togemon">Togemon</option>
+            <option value="WereGarurumon">WereGarurumon</option>
+            <option value="Zudomon">Zudomon</option>
+          </select>
+          <br>
+          <br>
+          <fieldset>
+            <legend>Terminos</legend>
+            <label>He leído los datos</label>
+            <input type="checkbox">
```

Comment:
What is missing in this label and input?

Reply:
For and id